### PR TITLE
Added news for Kubernetes 1.28 update

### LINKED ISSUE #4746: [News] Kubernetes 1.28 improves open-source cloud-native compute and networking
We're in the news! Using the contributing instructions for [adding news](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#adding-news), please add the following article to the [Layer5 Newsroom](https://layer5.io/news).

**Article:**
- Title: Kubernetes 1.28 improves open source cloud native compute and networking
- Link: https://www.sdxcentral.com/articles/news/kubernetes-1-28-improves-open-source-cloud-native-compute-and-networking/2023/08/

---
<h2>Contributor Resources and <a href="https://layer5.io/community/handbook">Handbook</a></h2>

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- 📚 See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md).
- 🎨 Wireframes and [designs for Layer5 site](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](https://slack.layer5.io).

Join the Layer5 Community by submitting your [community member form](https://layer5.io/newcomer).

### PR DESCRIPTION
**Pull request by Ghat0tkach**
**Description**

This PR fixes #4746 

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: src/collections/news/2023/2023-08-16-kubernetes-1.28-improves-open-source-cloud-native-compute-and-networking/index.mdx [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+import { Link } from "gatsby";
+
+<NewsWrapper>
+
+The open-source Kubernetes cloud-native platform is out with its second major update of 2023, introducing a long list of enhancements for operators in the new 1.28 release.
+
+Kubernetes is an open-source project, originally started by Google and now developed under the Linux Foundation’s Cloud Native Computing Foundation (CNCF), benefiting from the contributions of more than 900 companies. Among the updates in Kubernetes 1.28 are enhancements designed to help with resiliency, including the ability to recover from non-graceful node shutdowns. There are also a series of networking-related enhancements that will enable better security and performance.
+
+As is the case with every Kubernetes release, there is a theme and a code name; for Kubernetes 1.28 the name is “Planternetes.” “So in the northern hemisphere, it’s summer right now and I think the symbolism of the garden and our community goes hand in hand,” Grace Nguyen, release lead for Kubernetes 1.28, told SDxCentral. “Each of us has a very important role in the ecosystem and together we built this big, open-source project that has a lot of impact.”
+
```
Comment (Chadha93) [inline src/collections/news/2023/2023-08-16-kubernetes-1.28-improves-open-source-cloud-native-compute-and-networking/index.mdx:24]: @Ghat0tkach You missed adding the most essential section which mentions Meshery, layer, and @leecalcote. 

image

Comment (Ghat0tkach) [reply src/collections/news/2023/2023-08-16-kubernetes-1.28-improves-open-source-cloud-native-compute-and-networking/index.mdx:24]: @Chadha93 Really apologies , Shall I add link to meshery and member profile of Lee as well?

Comment (Chadha93) [reply src/collections/news/2023/2023-08-16-kubernetes-1.28-improves-open-source-cloud-native-compute-and-networking/index.mdx:24]: Yes.

Comment (Ghat0tkach) [reply src/collections/news/2023/2023-08-16-kubernetes-1.28-improves-open-source-cloud-native-compute-and-networking/index.mdx:24]: on my way

File: src/collections/news/2023/2023-08-16-kubernetes-1.28-improves-open-source-cloud-native-compute-and-networking/index.mdx [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+“The latest release of Kubernetes has a number of enhancements to help increase the stability, performance and maintainability of the core platform while also augmenting workload innovation, including AI [artificial intelligence] and virtual machines,” Karena Angell, OpenShift Commons lead and senior principal product manager for Red Hat OpenShift and Hybrid Platforms, told SDxCentral.
+
+Angell said Red Hat is particularly interested in the new node system memory swap support that has been added to Kubernetes 1.28 to enable better memory performance across a cluster. Additionally, she noted that the Kubernetes Job API now allows for more choices in AI model training and retraining, which is especially key for AI-driven and intelligent workloads.
+
+“As Kubernetes continues to mature as a platform with release 1.28, we’re very happy to see a number of new features and tweaks that further support production stability and consistency,” Angell said. “Large and complicated jobs can now fail faster and more accurately which, while it sounds strange, contributes to better overall performance of Kubernetes in production.”
```
Comment (Chadha93) [inline src/collections/news/2023/2023-08-16-kubernetes-1.28-improves-open-source-cloud-native-compute-and-networking/index.mdx:34]: @Ghat0tkach Please move this section down, and bring the "Networking enhancements in Kubernetes 1.28" up.

Comment (Ghat0tkach) [reply src/collections/news/2023/2023-08-16-kubernetes-1.28-improves-open-source-cloud-native-compute-and-networking/index.mdx:34]: Yes doing the same

**Comment by l5io** [comment]
🚀 Preview for commit c994ba779c6c2595665596d5fa8da323f26d2836 at: https://64dde6c5503d2664e750c748--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit e7b05275a4803bb52ffab29af95eb5b6cc539bb6 at: https://64de7041812ffb0469b446a0--layer5.netlify.app

**Review by Chadha93 — APPROVED** [review]

**Pull request merged**